src/main/java/dp/handler/Handler.java

```diff
@@ -330,20 +331,8 @@ private WorkbookDetails createWorkbook(S3Object object, Metadata datasetMetadata
                 if (isPublished) {
                     s3Client.putObject(putObjectRequest);
                 } else {
-                    byte[] psk = createPSK();
-
-                    String path = vaultPath + "/" + Paths.get(filename).getFileName().toString();
-                    String vaultKey = "key";
-
-                    Map<String, Object> map = new HashMap<>();
-                    map.put(vaultKey, Hex.encodeHexString(psk));
-
-                    info().path(path).log("writing key to vault");
-
-                    vaultOperations.write(path, map);
                     putObjectRequest.setBucketName(privateBucket);
-
-                    s3Crypto.putObjectWithPSK(putObjectRequest, psk);
+                    s3Crypto.putObject(putObjectRequest);
```

Comment:
The s3Crypto object is using [an extended version of the standard S3 client](https://github.com/ONSdigital/java-s3crypto) (see line 102) with added psk functionality. We don't need this anymore so we should really just use a standard S3 client instead. 
(…and also reconsider the name of it)

Reply:
Being as the name of the bucket is passed in to the request, I have used the standard s3client as this should direct the request to the private bucket in this instance.